etl/grapher_helpers.py

```diff
@@ -782,7 +782,7 @@ def adapt_table_with_dates_to_grapher(
         tb = tb.drop(columns=["year"], errors="raise")
 
     # Ensure date column is in datetime format.
-    tb[date_column] = pd.to_datetime(tb[date_column])
+    tb[date_column] = pd.to_datetime(tb[date_column].astype(object))
```

Comment:
maybe can use `astype("string")`?

Reply:
I don't recall why, but it wasn't working for some reason.